parser/charset/charset.go

```diff
@@ -193,33 +197,35 @@ func GetCollationByID(id int) (*Collation, error) {
 }
 
 const (
-	// CharsetBin is used for marking binary charset.
-	CharsetBin = "binary"
 	// CollationBin is the default collation for CharsetBin.
 	CollationBin = "binary"
-	// CharsetUTF8 is the default charset for string types.
-	CharsetUTF8 = "utf8"
 	// CollationUTF8 is the default collation for CharsetUTF8.
 	CollationUTF8 = "utf8_bin"
-	// CharsetUTF8MB4 represents 4 bytes utf8, which works the same way as utf8 in Go.
-	CharsetUTF8MB4 = "utf8mb4"
 	// CollationUTF8MB4 is the default collation for CharsetUTF8MB4.
 	CollationUTF8MB4 = "utf8mb4_bin"
-	// CharsetASCII is a subset of UTF8.
-	CharsetASCII = "ascii"
 	// CollationASCII is the default collation for CharsetACSII.
 	CollationASCII = "ascii_bin"
-	// CharsetLatin1 is a single byte charset.
-	CharsetLatin1 = "latin1"
 	// CollationLatin1 is the default collation for CharsetLatin1.
 	CollationLatin1 = "latin1_bin"
 
 	CollationGBKBin       = "gbk_bin"
 	CollationGBKChineseCI = "gbk_chinese_ci"
+)
+
+const (
+	// CharsetASCII is a subset of UTF8.
+	CharsetASCII = "ascii"
+	// CharsetBin is used for marking binary charset.
+	CharsetBin = "binary"
```

Comment:
Can you explain the rename? It's not a huge deal, but there are some users using this as a library. Renaming `CharsetBinary` to `CharsetBin` might cause them some issues. But if `Bin` is more correct, I have no problem with it.

Reply:
We have `CharsetBin` and `CharsetBinary` in parser before. I found we use `CharsetBin` more, so I decided delete `CharsetBinary`.